File: firstcode/firstassignemt/newassignement/roadtax.py
def roadtax(carcost):

	if carcost < 1000000:
		car_roadtax = taxcalculator(carcost, 10)
		
		return car_roadtax
	elif carcost < 3000000:
		car_roadtax = taxcalculator(carcost, 15)
		
		return car_roadtax
	elif carcost < 5000000:
		car_roadtax = taxcalculator(carcost, 20)
		return car_roadtax
	else:
		car_roadtax = taxcalculator(carcost, 25)
		return car_roadtax

def taxcalculator(cost, percentage):
	percentageCost = (percentage/100)
	calculated_tax = (cost * percentageCost)
	
	return calculated_tax

car_type = input("What is the name of your car: ")

flag = ''
while flag  != "no":
	carcost = int(input("Enter the cost of your car:  "))
	amount_paid_on_road_tax = roadtax(carcost)
	
	print(f"the road tax to be paid based on your {car_type} car price is {amount_paid_on_road_tax}")
	
	flag = input("press yes to continue or no to end:  ")
	

File: firstcode/firstassignemt/myjavaassignment/roadtaxcode/RoadTax.java
import java.util.Scanner;
	
public class RoadTax {
	public static void main(String[] args) {
		System.out.println("Welcome friend, today must be a very good day\n");
	
	Scanner input = new Scanner(System.in);
	
	
	String flag = new String(" ");
	
	while(!flag.equals("no")) {

		String carname;
		System.out.print("What is you car name:  ");
			carname = input.next();


		TaxCalculator tax = new TaxCalculator();
		double carcost;

		System.out.println("Enter the cost of your car\n");
			carcost = input.nextDouble();

		double AmountPaidOnRoadTax = tax.roadtax(carcost);
		
		System.out.printf("the road tax to be paid based on your %s car price is %.2f %n", carname, AmountPaidOnRoadTax);
	
		System.out.println("press yes to continue or no to end");
			flag = input.next();
	}
}
}